.env.warden

```
@@ -0,0 +1,39 @@
+WARDEN_ENV_NAME=hackman
+WARDEN_ENV_TYPE=laravel
+WARDEN_WEB_ROOT=/
+
+TRAEFIK_DOMAIN=hackman.test
+TRAEFIK_SUBDOMAIN=app
+
+MYSQL_DISTRIBUTION=mariadb
+MYSQL_DISTRIBUTION_VERSION=10.4
+NODE_VERSION=12
+COMPOSER_VERSION=2
+PHP_VERSION=7.4
+PHP_XDEBUG_3=1
+REDIS_VERSION=5.0
+
+WARDEN_DB=1
+WARDEN_REDIS=1
+
+## Laravel Config
+APP_URL=http://app.hackman.test
+APP_KEY=base64:Q+XXtB5dsJcoodRVZxyF/f+nWDknjI4kCHxQS2xkbxY=
```

Best avoiding committing secrets to the repository, even if they're only for development environments. Better to have people generate their own via `php artisan key:generate`.

```suggestion
APP_KEY=
```